src/calculateRentalCost.js

```diff
@@ -28,7 +28,25 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_FOR_DAY = 40;
+
+  const MIDDLE_TERM = 3;
+  const MIDDLE_TERM_DISCONT = 20;
+
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCONT = 50;
+
+  if (days >= LONG_TERM) {
+    return RENT_FOR_DAY * days - LONG_TERM_DISCONT;
+  }
+
+  if (days >= MIDDLE_TERM && days < LONG_TERM_DISCONT) {
+    return RENT_FOR_DAY * days - MIDDLE_TERM_DISCONT;
+  }
+
+  return days * RENT_FOR_DAY;
```

Comment:
`days * RENT_FOR_DAY` is now executed three times.
This can be put into a separate variable and reused.